qemu/tests/cfg/virtio_mode.cfg

```diff
@@ -0,0 +1,143 @@
+- virtio_mode:
+    type = virtio_mode
+    cdroms += " virtio winutils"
+    kill_vm = yes
+    blk_extra_params_stg = "serial=TARGET_DISK0"
+    blk_extra_params_disk1 = "wwn=0x5000123456789abc"
+    fs_type = xfs
+    RHEL.6:
+        fs_type = ext4
+    format_cmd = "yes | mkfs.${fs_type} {0}"
+    show_dev_cmd = "ls {0}"
+    mount_cmd =  "mkdir -p /media && mount -t ${fs_type} {0} /media"
+    show_mount_cmd = "mount | grep {0}"
+    umount_cmd = "umount {0}"
+    testfile_name = "/media/format_disk-test.txt"
+    writefile_cmd = "echo %s > %s"
+    readfile_cmd = "cat %s"
+    iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 1G -g 2G -M -i 0 -i 1 -b iozone.xls -f ${disk_letter}:\testfile"
```

Comment:
1. No disk_letter param.
2. Should need format param if you want to run iozone test on data disk. format disk need first be executed.
3. Better reduce the param '-i 0 -i 1' if added '-a' param. use -i param will not trigger analyses function in iozone_windows, for automation test,  open the analyses function is better~